install/terraform/gcp/buckets_iam.tf

```diff
@@ -50,3 +42,30 @@ resource "google_storage_bucket_iam_member" "data_loader_datasets_storage_admin"
   member = "serviceAccount:${google_service_account.data_loader.email}"
 }
 
+# GCP manager #
+
+resource "google_storage_bucket_iam_member" "gcp_manager_models_storage_admin" {
+  bucket = google_storage_bucket.models.name
+  role   = "roles/storage.admin"
```

Comment:
For the moment, `roles/storage.admin` is over-permissioning but we've already identified use cases where storage.admin will eventually be needed anyway (model or dataset deletion, signed urls for other reasons upload or download)